src/app/screens/Book/components/Row/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { View, Text, Image } from 'react-native';
+
+import defaultImage from '../../../../../assets/defaultImage.png';
+
+import styles from './styles.js';
+
+class Row extends Component {
```

Comment:
this class could be declared as a function, because it only has the `render` method.